src/pykeen/training/training_loop.py

```diff
@@ -83,9 +84,20 @@ def _get_optimizer_kwargs(optimizer: Optimizer) -> Mapping[str, Any]:
     return optimizer_kwargs
 
 
+def _get_lr_scheduler_kwargs(lr_scheduler: Union[_LRScheduler, ReduceLROnPlateau]) -> Mapping[str, Any]:
+    lr_scheduler_kwargs = lr_scheduler.state_dict()
+    lr_scheduler_kwargs = {
+        key: value
+        for key, value in lr_scheduler_kwargs.items()
+        if key[0] != '_' and key not in ['base_lrs', 'last_epoch']
```

Comment:
not sure if `key[0] != '_'` or `not key.startswith('_')` is faster, but the latter might be a bit easier to read